nomos-services/network/src/backends/libp2p.rs

```diff
@@ -73,6 +74,15 @@ struct MixnetMessage {
     message: Box<[u8]>,
 }
 
+impl MixnetMessage {
+    pub fn as_bytes(&self) -> Box<[u8]> {
+        wire::serialize(self).unwrap().into_boxed_slice()
```

Comment:
Better to use `expect` with some message (it shouldn't happen anyway,but just in case)

Reply:
👍 e823545